nextflow_schema.json

```diff
@@ -0,0 +1,64 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/ebi-metagenomics/mobilome-annotation-pipeline/master/nextflow_schema.json",
+    "title": "ebi-metagenomics/mobilome-annotation-pipeline pipeline parameters",
+    "description": "The Mobilome Annotation Pipeline",
+    "type": "object",
+    "$defs": {
+        "input_output_options": {
+            "title": "Input/output options",
+            "type": "object",
+            "fa_icon": "fas fa-terminal",
+            "description": "Define where the pipeline should find input data and save output data.",
+            "properties": {
+                "input": {
```

Comment:
maybe rename that to `samplesheet` insted of input?

Reply:
Is there a convention? Other  pipelines use `--input` as well:
https://github.com/EBI-Metagenomics/mettannotator?tab=readme-ov-file#running-mettannotator
https://github.com/EBI-Metagenomics/amplicon-pipeline?tab=readme-ov-file#execution